src/taxContent/README.txt

```diff
@@ -1 +0,0 @@
-This folder is for storing offline tax content. The Avalara AvaTax C# SDK includes methods to enable offline, ZIP-only calculations for cases when your integration fails to receive a tax calculation response in the time you allotted. 
```

Comment:
I think we need to leave this in, its going to get regenerated by SDK generator.